### PR TITLE
Add description for action

### PR DESCRIPTION
**Pull request by colevohs**
Add description for action.

## Testing

- [X] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [X] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Comment by sayan-das-in** [comment]
Could you please update the Action name as well ?
It currently says `updateEmailContactProfileFields` which might be confusing for users but what you actually want to do is to update Contact List I think.

**Comment by sayan-das-in** [comment]
Let me know if this needs more time. We can deploy this next week.

**Comment by colevohs** [comment]
> Could you please update the Action name as well ? It currently says `updateEmailContactProfileFields` which might be confusing for users but what you actually want to do is to update Contact List I think.
> 
> For reference, some examples of action names are `Uspert Contact`, `Track User Action`, `Record Purchase`, etc.

The action name is correct for the API route it corresponds to. It does not add or remove contacts from the list. It updates the metadata on the contact. See the route here: https://api.listrak.com/email#operation/ContactSegmentationField_PostContactSegmentationFieldValuesResource

**Comment by sayan-das-in** [comment]
Thanks for the clarification.

**Review by sayan-das-in — APPROVED** [review]

File: packages/destination-actions/src/destinations/listrak/updateEmailContactProfileFields/index.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ type RequestsByListId = { [k: number]: ContactSegmentationFieldValues[] }
 
 const action: ActionDefinition<Settings, Payload> = {
   title: 'Update Email Contact Profile Fields',
```
[the remark this answers is not in the document]
Comment (gary-hopkins-listrak) [reply packages/destination-actions/src/destinations/listrak/updateEmailContactProfileFields/index.ts:19]: @sayan-das-in in our API documentation the title of the route is "Start a list import to update one or more contact profile fields". Would that be acceptable?

[the remark this answers is not in the document]
Comment (sayan-das-in) [reply packages/destination-actions/src/destinations/listrak/updateEmailContactProfileFields/index.ts:19]: What I would do for now is merge the changes as-is in this PR, and we can work out any changes later. I would really have to publish the update as this is the only item holding off this week's release.

[the remark this answers is not in the document]
Comment (gary-hopkins-listrak) [reply packages/destination-actions/src/destinations/listrak/updateEmailContactProfileFields/index.ts:19]: @sayan-das-in merging it as-is works for us. Thanks!

**Pull request merged**